README.md

```diff
@@ -54,76 +54,75 @@ The project is primarily composed of two areas:
+To emulate Azure storage services (blobs, tables and queues) you will require one of the following options.
+   - [Azurite for Docker and Docker Compose](https://docs.docker.com/) - recommended, see [Setting up the storage emulator](#setting-up-the-database-and-storage-emulator)
+   - Alternatively, if opting to not use Storage Explorer at all, you could create your own Storage
+     Account on Azure and amend your storage connection strings to point to this.
+     - [Azure Storage Account](https://azure.microsoft.com/en-gb/services/storage/)
+     - [Running against other databases](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator#start-and-initialize-the-storage-emulator)
 
-     - Alternatively, if opting to not use Storage Explorer at all, you could create your own Storage
-       Account on Azure and amend your storage connection strings to point to this.
-       - [Azure Storage Account](https://azure.microsoft.com/en-gb/services/storage/)
-       - [Running against other databases](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator#start-and-initialize-the-storage-emulator)
+### Setup libmagic (Linux and Mac only)
 
+#### Linux only 
 
-4. Setup libmagic (Linux and Mac only)
+Add symlinks to `libmagic-1`:
 
-**Linux only** 
-
-  Add symlinks to libmagic-1:
-
-   ```sh
-   cd /usr/lib/x86_64-linux-gnu/
-   sudo ln -s libmagic.so.1.0.0 libmagic-1.so
-   sudo ln -s libmagic.so.1.0.0 libmagic-1.so.1
-   ```
-
-   See [bug raised with the library](https://github.com/hey-red/Mime/issues/36) for more info.
-   
-   Use correct version of `magic.mgc` file (Ubuntu 22.04 or later only):
+```sh
+cd /usr/lib/x86_64-linux-gnu/
+sudo ln -s libmagic.so.1.0.0 libmagic-1.so
+sudo ln -s libmagic.so.1.0.0 libmagic-1.so.1
+```
 
-   If you're using Ubuntu 22.04 or later you'll need to place an updated [magic.mgc](https://github.com/hey-red/Mime/blob/master/src/Mime/content/magic.mgc) file in `src/GovUk.Education.ExploreEducationStatistics.Common/libs` due to an issue with mismatched versions of `libmagic` and `magic.mgc`. You will then need to rebuild the project to ensure the new file is copied to the output directory.
+See [bug raised with the library](https://github.com/hey-red/Mime/issues/36) for more info.
 
-**Mac only**
+Fix `magic.mgc` file version mismatches:
 
-   Install and link libmagic:
+If you're using Ubuntu 22.04, or later you'll most likely be using a `libmagic-1` binary that isn't 
+synchronized correctly with the `magic.mgc` file distributed with the build. This results in the 
+processor service failing during validation.
 
-    ```sh
-    brew install libmagic
-    brew link libmagic
-    env ARCHFLAGS="-arch x86_64" sudo gem install ruby-filemagic -- --with-magic-include=/usr/local/include --with-magic-lib=/usr/local/lib/
-    ```
+To fix this, you need to create an `appsettings.Local.json` file in the relevant project and ensure
```

Comment:
I'd mention here specifically which projects a new starter would need to include this under, as it wouldn't be obvious to someone setting up for the first time.

Reply:
Cool ta